Formula/vespa-cli.rb

```diff
@@ -0,0 +1,28 @@
+class VespaCli < Formula
+  desc "Command-line tool for Vespa.ai"
+  homepage "https://vespa.ai"
+  url "https://github.com/vespa-engine/vespa/archive/refs/tags/vespa-7.462.20-1.tar.gz"
+  version "7.462.20"
+  sha256 "87e309fbdac0bef174ef3ac1bf094d551a54dd2da57ecc52e19880d0d81a9cda"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    cd "client/go" do
+      with_env(VERSION: version.to_s) do
+        system "make", "all", "manpages"
+      end
+      bin.install "bin/vespa"
+      man1.install Dir["share/man/man1/vespa*.1"]
+      (bash_completion/"vespa").write `#{bin}/vespa completion bash`
+      (fish_completion/"vespa.fish").write `#{bin}/vespa completion fish`
+      (zsh_completion/"_vespa").write `#{bin}/vespa completion zsh`
+    end
+  end
+
+  test do
+    assert_match "vespa version #{version}", shell_output("#{bin}/vespa version")
+    assert_match "Deploy (prepare and activate) an application package.", shell_output("#{bin}/vespa deploy -h")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
The problem is that this is just a client. You need to have a running Vespa installation for it to do something useful.

Reply:
I suppose I can make it change a config option (`vespa config`) and read it back, would that be sufficient?

Reply:
Can we try to access a non-existent server and check we get the expected error?

Checking the config would be good too.

Reply:
PTAL